oaknational/Oak-Web-Application
# Feat/pupil 894/quiz result short

## Description

Music year: 1951

- List of changes
OWA: QuizResultShortAnswer

## Issue(s)

Fixes #

## How to test

1. Go to storybook 
2. click onQuizResultShortAnswer
4. You should see component states

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
